packages/runtime/test/src/interpreter/pooling-inner-transactions-test.ts

```diff
@@ -0,0 +1,375 @@
+            TXN_OBJ.apat.push(Buffer.from(decodeAddress(applicationAccount.address).publicKey));
+            TXN_OBJ.apat.push(Buffer.from(decodeAddress(bobAccount.address).publicKey));
+
+            if (elonAcc) {
+                elonAcc.amount = 1000000n;
+                const assetID = interpreter.runtime.ctx.deployASADef(
+                    "test-asa",
+                    { total: 10, decimals: 0, unitName: "TASA" },
+                    elonAddr,
+                    { creator: { ...elonAcc.account, name: "elon" } }
+                ).assetIndex;
+
+                bobAccount.account.rekeyTo(applicationAccount.address);
+
+                TXN_OBJ.apas = [assetID];
+
+                assert.equal(interpreter.runtime.ctx.state.accounts
+                    .get(johnAcc.address)?.balance()
+                    , 0n);
+
+                const prog = `
+                itxn_begin
+                int axfer
+                itxn_field TypeEnum
+                global CurrentApplicationAddress
+                itxn_field AssetReceiver
+                int ${assetID}
+                itxn_field XferAsset
+                int 0
+                itxn_field AssetAmount
+                int 0
+                itxn_field Fee
+                itxn_next
+                int pay
+                itxn_field TypeEnum
+                global CurrentApplicationAddress
+                itxn_field Receiver
+                addr ${bobAccount.address}
+                itxn_field Sender
+                int 1000
+                itxn_field Amount
+                int 2000
+                itxn_field Fee
+                itxn_submit
+                int 1
+                return
+                `;
+
+                executeTEAL(prog);
+
+                assert.equal(interpreter.runtime.ctx.state.accounts
+                    .get(bobAccount.address)?.balance()
+                    , 1000000n - 3000n);
+                assert.equal(interpreter.runtime.ctx.state.accounts
+                    .get(johnAcc.address)?.balance()
+                    , 0n);
+                assert.equal(interpreter.runtime.ctx.state.accounts
+                    .get(appAccAddr)?.balance()
+                    , 1000n);
+                assert.isDefined(interpreter.runtime.ctx.state.accounts
+                    .get(appAccAddr)?.getAssetHolding(assetID));
+
+            }
+
+        });
+
+        it("Should succeed: when funded account partially covers it's own fee", function () {
+            assert.equal(interpreter.runtime.ctx.state.accounts
+                .get(johnAcc.address)?.balance()
+                , 0n);
+
+            const prog = `
+				itxn_begin
+				int pay
+				itxn_field TypeEnum
+				txn Applications 2
+				app_params_get AppAddress
+				assert
+				itxn_field Receiver
+				int 0
+				itxn_field Amount
+				//partially covering it's own fee
+				int 1
+				itxn_field Fee
+                itxn_next
```

Comment:
formating

Reply:
It just appears here but the format is good on vs code.

Reply:
the itxn_next is moved to left isn't it?

Reply:
It just appears on left only on GitHub but not vs code. I have faced this issue before. You can check in to my branch and check the formatting. It's not left aligned.

Reply:
ok, resolving